pages/management/applications.md

```diff
@@ -66,6 +66,40 @@ When you create a view, you are saving a specific set of filters that you may wa
 
 You will be asked to choose a name for the saved view. This view can then be selected at any time by clicking the *Views* button above the filter box.
 
+## Device tags
+
+Tags provide an easy way to include more information about your devices than what is already provided in the dashboard. A device can be given multiple tags, and the tags can be used when you create filters.
+
+### Create and edit tags
+
+To add a tag from the device list, click the checkbox to the left of the devices you wish to tag, followed by the *Tags* button on the right side of the dashboard:
+
+<img src="/img/common/app/tags.png" width="80%">
+
+This opens the tag management window. From here you can define a name for your tag and a value for this specific device. For example, you could create a tag to keep track of device location:
+
+<img src="/img/common/app/add_tag.png" width="80%">
+
+Tags you add to your device will appear in a list in the management window. They can be edited or removed from here:
+
+<img src="/img/common/app/tag_list.png" width="80%">
+
+When you are finished creating and editing tags, click *Apply* to keep the changes.
```

Comment:
How about `persist` instead of `keep`?
It might worth mentioning the `Add Tag` button as a reminder.